test.js

```diff
@@ -1655,6 +1655,16 @@ test('poetry py3.7 fails packaging if poetry.lock is missing and flag requirePoe
   t.end();
 });
 
+test('works with provider.runtime not being python', async (t) => {
+  console.log('im working');
```

Comment:
You might want to remove that `console.log` here as it might be messing up the test